public/app/plugins/panel/canvas/InlineEdit.tsx

```diff
@@ -11,19 +11,23 @@ import { InlineEditBody } from './InlineEditBody';
 
 type Props = {
   onClose?: () => void;
+  id: number;
 };
 
-const OFFSET_X = 70;
+const OFFSET_X = 10;
+const OFFSET_Y = 32;
 
-export const InlineEdit = ({ onClose }: Props) => {
-  const btnInlineEdit = document.querySelector('[data-btninlineedit]')!.getBoundingClientRect();
+export const InlineEdit = ({ onClose, id }: Props) => {
+  const parentPanel = document.querySelector(`[data-panelid="${id}"]`)!.getBoundingClientRect();
```

Comment:
This worries me a little bit as this may break due to unrelated changes over time (we change our class names etc)

I'm thinking it should be possible to just access the root element's information for this purpose (via `scene.root.div` property (root element is always 100% height / width of panel). That way we are not relying on structure of how we define DOM classes for this functionality

See `InlineEditBody.tsx` for getting the current active panel's instance state which will have the `scene` property on it. Maybe we can refactor that out of the body component and pass instance state down as a prop from `InlineEdit` into `InlineEditBody` to reduce number of observables (we can pair on this)